### PR TITLE
COMP: Set CMake Policy 135 to NEW to suppress warnings in CMake 3.24

### PR DESCRIPTION
**Pull request by dzenanz**
Prevents warnings.

## PR Checklist
- [x] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [x] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)

### CODE REVIEW

**Comment by dzenanz** [comment]
@jhlegarreta I think this will resolve the warning which appears in #3554.

**Review by jhlegarreta — APPROVED** [review]

**Comment by jhlegarreta** [comment]
> @jhlegarreta I think this will resolve the warning which appears in https://github.com/InsightSoftwareConsortium/ITK/pull/3554.

Will rebase the PR on `master` once this one merged.

**Comment by dzenanz** [comment]
I clicked merge button 10 seconds before you wrote this :(

**Pull request merged**
